### PR TITLE
Add "inspect network request" to Replay Passport

### PR DESCRIPTION
**Pull request by jonbell-lot23**
Note to Brian:

This was pretty straight-forward code, then I was working through some strange errors and may have ended up doing some crazy stuff to fix them. Thanks for taking a look!

### CODE REVIEW

File: src/ui/components/NetworkMonitor/RequestRow.tsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ export const RequestRow = ({
   isInLoadedRegion: boolean;
   isInPast: boolean;
   isSelected: boolean;
-  onClick: (row: RequestSummary) => void;
+  onClick: (row: RequestSummary, event: React.MouseEvent) => void;
```
Comment (bvaughn) [inline src/ui/components/NetworkMonitor/RequestRow.tsx:25]: Why did this change?

File: src/ui/components/NetworkMonitor/RequestTable.tsx [hunk 2]
```diff
@@ -39,11 +41,19 @@ const RequestTable = ({
...
     trackEvent("net_monitor.seek_to_request");
     seek(request.point.point, request.point.time, true);
-    onRowSelect(request);
+    const syntheticEvent = new MouseEvent("click", {
+      bubbles: true,
+      cancelable: true,
+      view: window,
+      button: 0,
+    });
+    onRowSelect(request, syntheticEvent as unknown as React.MouseEvent<Element, MouseEvent>);
```
Comment (bvaughn) [inline src/ui/components/NetworkMonitor/RequestTable.tsx:50]: This change seems unnecessary (the addition of the `event` prop)

File: src/ui/components/Passport/Passport.module.css [hunk 3]
```diff
@@ -1,8 +1,8 @@
 .PassportBoxWrapper {
   position: relative;
-  background-color: powderblue;
+  background-color: #dde7ef;
```
Comment (bvaughn) [inline src/ui/components/Passport/Passport.module.css:3]: Hard-coded colors seem like they'd break our light/dark theme?

**Review by bvaughn — CHANGES_REQUESTED** [review]

File: src/ui/components/Passport/Passport.module.css (hunk 3 above)
Comment (jonbell-lot23) [reply src/ui/components/Passport/Passport.module.css:3]: Currently the passport is always the same blue color with passport pattern.

File: src/ui/components/NetworkMonitor/RequestTable.tsx (hunk 2 above)
Comment (jonbell-lot23) [reply src/ui/components/NetworkMonitor/RequestTable.tsx:50]: Yeah this was the strange stuff I was referring to. When I wrote the code without this, I was getting tdc errors, and it was only by adding this odd stuff that I could get it working.

I can take another look but this is beyond my understanding. Might need to reach out and ask for some help if/when I get the same tsc errors.

Comment (jonbell-lot23) [reply src/ui/components/NetworkMonitor/RequestTable.tsx]: Ugh, so embarrassing. I figured out why I ended up on that wild goose chase and reverted. What a dumb bit of code that was!

File: src/ui/components/NetworkMonitor/RequestRow.tsx (hunk 1 above)
Comment (jonbell-lot23) [reply src/ui/components/NetworkMonitor/RequestRow.tsx:25]: Stupidity, but fixed now

**Comment by jonbell-lot23** [comment]
Ok! Fixed and should be ready to go?

File: src/ui/components/NetworkMonitor/RequestTable.tsx [hunk 4]
```diff
@@ -79,7 +83,11 @@ const RequestTable = ({
                 isInPast={inPast}
                 isSelected={selectedRequest?.id === row.original.id}
                 key={row.getRowProps().key}
-                onClick={onRowSelect}
+                onClick={request => {
+                  dismissInspectNetworkRequestNag();
+                  console.log("calling");
```
Comment (markerikson) [inline src/ui/components/NetworkMonitor/RequestTable.tsx:88]: :hand: Remove dead logging

**Review by markerikson — COMMENTED** [review]
Fix the one `console.log()` and then this looks good

File: src/ui/components/NetworkMonitor/RequestTable.tsx (hunk 2 above)
Comment (bvaughn) [reply src/ui/components/NetworkMonitor/RequestTable.tsx]: No worries. We all do stuff like that. :)

File: src/ui/components/NetworkMonitor/RequestRow.tsx (hunk 1 above)
Comment (bvaughn) [inline src/ui/components/NetworkMonitor/RequestRow.tsx:25]: This change still seems unnecessary. Nothing is using the event param, is it?

File: src/ui/components/NetworkMonitor/RequestRow.tsx [hunk 5]
```diff
@@ -51,7 +51,7 @@ export const RequestRow = ({
           [styles.future]: !isInPast,
           [styles.unloaded]: !isInLoadedRegion,
         })}
-        onClick={() => onClick(row.original)}
+        onClick={event => onClick(row.original, event)}
```
Comment (bvaughn) [inline src/ui/components/NetworkMonitor/RequestRow.tsx:54]: This change still seems unnecessary. Nothing is using the event param, is it?

Comment (jonbell-lot23) [reply src/ui/components/NetworkMonitor/RequestRow.tsx:54]: Sorry, missed that one. Removing

File: src/ui/components/NetworkMonitor/RequestRow.tsx (hunk 1 above)
Comment (jonbell-lot23) [reply src/ui/components/NetworkMonitor/RequestRow.tsx]: Mistake, removed

**Comment by jonbell-lot23** [comment]
Ok, *now* I think it's good to go. TSC fine, lint fine, all previously mentioned issues seem fine. Thanks again for your help!

**Review by bvaughn — APPROVED** [review]

**Pull request merged**